app/controllers/api/v2/profiles_controller.rb

```diff
@@ -38,18 +44,26 @@ def profile_params
         params.require(:data).permit(PROFILE_ATTRIBUTES).to_h
       end
 
-      def profile_attributes
+      def new_profile_attributes
```

Comment:
I guess we'll not separate attributes like this later?

Reply:
yea as when we revisit this we won't need the first name/last name